src/shared/cloudformation/activation.ts

```diff
@@ -10,7 +10,8 @@ import { localize } from '../utilities/vsCodeUtils'
 import { CloudFormationTemplateRegistry } from './templateRegistry'
 import { CloudFormationTemplateRegistryManager } from './templateRegistryManager'
 
-export const TEMPLATE_FILE_GLOB_PATTERN = '**/template.{yaml,yml}'
+// ** VSCode glob doesn't expand the first **, so it also needs a second **
+export const TEMPLATE_FILE_GLOB_PATTERN = '{!(.aws-sam),**}/**/template.{yaml,yml}'
```

Comment:
Comment is confusing. If it doesn't expand the first `**` then what is the purpose of the first `**` ?

Reply:
it does, but not in the way that you would expect. it doesn't match deeper directories

Reply:
Ah maybe "recursively expand" is better?

Reply:
Message updated